packages/cli/src/TagHelpers.ts

```diff
@@ -38,6 +38,8 @@ export async function getTagsWithCountDb(tablePrefix: string): Promise<ITagWithC
 		.createQueryBuilder()
 		.select(`${tablePrefix}tag_entity.id`, 'id')
 		.addSelect(`${tablePrefix}tag_entity.name`, 'name')
+		.addSelect(`${tablePrefix}tag_entity.createdAt`, 'createdAt')
+		.addSelect(`${tablePrefix}tag_entity.updatedAt`, 'updatedAt')
```

Comment:
Wondering if it'd make sense to just select `*` instead of all the columns individually. Same for `GET /tags` in `Server.ts`.

Reply:
being specific does not hurt. Also don't have to worry when adding columns that should not make it to FE